cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/bungee/command/CommandHub.java

```diff
@@ -10,8 +10,10 @@
 
 public final class CommandHub extends Command {
 
+    String[] aliases = BridgeConfigurationProvider.load().getLobbyCommandAliases().toArray(new String[0]);
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/CloudNetService/CloudNet-v3/pullRequest?prid=5780263)